### PR TITLE
Please merge this

### PR DESCRIPTION
**Pull request by vfurmane**

### CODE REVIEW

**Comment by vfurmane** [comment]
Merge after #89

File: include/commands.hpp [hunk 1]
```diff
@@ -43,6 +43,8 @@ struct Dependencies
 _base_channel				parseChannel(const std::string &channel);
 std::vector<std::string>	parseList(const std::string &list);
 
+
```
Comment (ppiques) [inline include/commands.hpp:46]: extra newline

**Review by ppiques — CHANGES_REQUESTED** [review]

File: include/commands.hpp (hunk 1 above)
Comment (theovgl) [inline include/commands.hpp:46]: ???

**Review by theovgl — CHANGES_REQUESTED** [review]

**Review by theovgl — APPROVED** [review]

**Review by ppiques — APPROVED** [review]
gpalu

**Pull request merged**